dll/win32/kernel32/client/thread.c

```diff
@@ -930,12 +930,16 @@ GetThreadId(IN HANDLE Thread)
 }
 
 /*
- * @unimplemented
+ * @implemented
  */
 LANGID
 WINAPI
 SetThreadUILanguage(IN LANGID LangId)
```

Comment:
```suggestion
SetThreadUILanguage(
    _In_ LANGID LangId)
```
SAL2 is preferred.

Reply:
So the "implemented" thing would have to be described properly per SAL2.